src/main/java/school/hei/haapi/endpoint/rest/mapper/UserMapper.java

```diff
@@ -90,12 +93,34 @@ public User toDomain(Student student) {
         .lastName(student.getLastName())
         .email(student.getEmail())
         .ref(student.getRef())
-        .status(User.Status.fromValue(student.getStatus().toString()))
+        .status(toDomainUserStatus(student.getStatus()))
         .phone(student.getPhone())
         .entranceDatetime(student.getEntranceDatetime())
         .birthDate(student.getBirthDate())
         .sex(student.getSex() == null ? null : User.Sex.valueOf(student.getSex().toString()))
         .address(student.getAddress())
         .build();
   }
+
+  private User.Status toDomainUserStatus(EnableStatus status) {
+    switch (status) {
+      case ENABLED:
+        return User.Status.ENABLED;
+      case DISABLED:
+        return User.Status.DISABLED;
+      default: throw new BadRequestException("Unexpected type: " + status);
+    }
+  }
```

Comment:
if suspended throw not implemented

also use new switch case form if permitted : 
`switch(var){
case SOMETHING -> SOMETHING_ELSE
   }
`